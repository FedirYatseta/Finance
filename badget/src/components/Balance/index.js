import { Wrapper } from "../Transaction/style";

const Balance = ({ balance }) => {
    return (
        <div>
            <Wrapper value={balance}>
                Balance: {balance}
            </Wrapper>

        </div>
    )
}

export default Balance;