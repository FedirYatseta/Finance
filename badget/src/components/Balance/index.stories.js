import React from 'react';

import Balance from '.';

export default {
    title: 'Example/Balance',
    component: Balance,
    argTypes: {
        variant: {
            options: ['primary', 'secondary'],
        },
    },
};

const Template = (args) => <Balance {...args} />;

export const Income = Template.bind({});
Income.args = {
    variant: 'primary',

};