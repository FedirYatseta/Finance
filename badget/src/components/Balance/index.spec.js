

//import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import Balance from '.';



describe('Balance', () => {
  it('should show Balance', () => {
    const balance = 15
    const component = renderer.create(
      <Balance balance={balance} />,
    );
    expect(component)

  })
});