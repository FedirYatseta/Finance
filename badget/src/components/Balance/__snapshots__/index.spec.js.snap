// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Balance should show Balance 1`] = `
<div>
  <div
    className="sc-bczRLJ cKPmbC"
  >
    Balance: 
  </div>
</div>
`;
